.vsts-ci.yml

```diff
@@ -8,59 +11,70 @@ resources:
     ref: refs/heads/master
 
 variables:
-  ${{ if in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'PullRequest') }}:
+  teamName: Roslyn-Project-System
```

Comment:
I assume this is correct and you guys are sharing a TeamName?

Reply:
Yeap.